### ISSUE TITLE
Update submitter guide to use v4.1 schema

### ISSUE THREAD

**Issue by spbnick** [kernelci/kcidb#375]
Now that we've switched to using the v4.1 schema in our deployment, update the [submitter guide](https://github.com/kernelci/kcidb/blob/main/doc/submitter_guide.md) to use it in examples, and describe the new object types.

**Comment by NikkyXO** [comment]
hello @spbnick i will like to work on this

**Comment by spbnick** [comment]
Absolutely! For this you will need to know the differences between the currently-described [v4.0](https://github.com/kernelci/kcidb-io/blob/main/kcidb_io/schema/v04_00.py) schema and the newly-deployed [v4.1](https://github.com/kernelci/kcidb-io/blob/main/kcidb_io/schema/v04_01.py), which basically amount to the addition of `issues` and `incidents`. For the start you can work on just updating the version numbers in the samples and the text (if any), and we could go from there :relaxed:

**Comment by NikkyXO** [comment]
> Absolutely! For this you will need to know the differences between the currently-described [v4.0](https://github.com/kernelci/kcidb-io/blob/main/kcidb_io/schema/v04_00.py) schema and the newly-deployed [v4.1](https://github.com/kernelci/kcidb-io/blob/main/kcidb_io/schema/v04_01.py), which basically amount to the addition of `issues` and `incidents`. For the start you can work on just updating the version numbers in the samples and the text (if any), and we could go from there relaxed

Alrighnt, well noted

**Comment by spbnick** [comment]
@NikkyXO updated the schema versions in the guide, but we still need sections on "issues" and "incidents" added.

**Comment by NikkyXO** [comment]
I could work on adding the "issues" and "incidents"

**Comment by spbnick** [comment]
Sure, @NikkyXO, go ahead! You'll need to understand better what each object in the schema does and what is the intention behind them. Look at the JSON schema generated by `kcidb-schema`, it contains some minimal documentation. You could also take a look at a related presentation I made last year: https://archive.fosdem.org/2022/schedule/event/masking_known_issues_across_six_kernel_ci_systems/ - it might give you some background.

**Comment by NikkyXO** [comment]
> Sure, @NikkyXO, go ahead! You'll need to understand better what each object in the schema does and what is the intention behind them. Look at the JSON schema generated by `kcidb-schema`, it contains some minimal documentation. You could also take a look at a related presentation I made last year: https://archive.fosdem.org/2022/schedule/event/masking_known_issues_across_six_kernel_ci_systems/ - it might give you some background.

Alright @spbnick
